src/commands/plugins/trust/verify.ts

```diff
@@ -50,6 +50,7 @@ export class Verify extends SfdxCommand {
       cacheDir: get(this.config, 'configDir') as string,
       configDir: get(this.config, 'cacheDir') as string,
       dataDir: get(this.config, 'dataDir') as string,
+      rootDir: get(this.config, 'root') as string,
```

Comment:
We should rename `rootDir` to something more semantic, like `cliRoot`, when it's passed to the verifier. When we're elsewhere in the code it'll be hard to tell what we are referring to.

This comment applies to everywhere `root` or `rootDir` is being used in this PR. 
The "what" in the question "the root of what?" should be clear regardless of where in the code this value is present.